.pylintrc

```diff
@@ -52,12 +52,12 @@ ignore=CVS
 # ignore-list. The regex matches against paths and can be in Posix or Windows
 # format. Because '\\' represents the directory delimiter on Windows systems,
 # it can't be used as an escape character.
-ignore-paths=
+ignore-paths=picard\\ui\\forms|picard/ui/forms
```

Comment:
As I understand it you don't need to specify both Windows and Unix path delimiters. The pattern will be interpreted as a path regardless.

Reply:
This was generated by `pylint --generate-rcfile` but I just noticed it will do weird things if used again:

```
pylint --generate-rcfile |pylint --generate-rcfile|grep ignore-paths   
ignore-paths=picard\\ui\\forms|picard\\ui\\forms|picard/ui/forms|picard/ui/forms
```

I'd say that's a pylint bug...


I'll revert back to just `picard/ui/forms`

Reply:
Fixed and rebased in https://github.com/metabrainz/picard/pull/2475/commits/07727e367e92e3ca19d0bb64ad637fdb4a011e27